src/game_battlealgorithm.cpp

```diff
@@ -62,9 +62,12 @@ Game_BattleAlgorithm::AlgorithmBase::AlgorithmBase(Type ty, Game_Battler* source
 	type(ty), source(source), targets(std::move(in_targets))
 {
 	assert(source != nullptr);
+
+#ifndef NDEBUG
```

Comment:
Does it nag about an empty loop here?
Because asserts are no-ops in release builds anyway, so should be optimized away.

Reply:
It nags about the unused t loop var when the assert is removed.